relational_operators/CreateIndexOperator.hpp

```diff
@@ -75,7 +83,8 @@ class CreateIndexOperator : public RelationalOperator {
 
  private:
   CatalogRelation *relation_;
-  const std::string &index_name_;
+  std::string index_name_;
```

Comment:
@zuyu Looks like there is a single query plan pointer that is maintained and is passed to the Foreman, so the references will never change. Thanks for pointing this out. I will update this.